src/client/pages/LoginPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box, Flex, Tab, TabList, TabPanel, TabPanels, Tabs } from '@chakra-ui/react';
+import { LoginForm } from '../components/LoginForm';
+import { RegisterForm } from '../components/RegisterForm';
+
+export const LoginPage = () => {
+  return (
+    <Flex justifyContent="center" align="center" bgColor="#75b7ea" h="100vh">
```

Comment:
when its a `<Flex>` u can just use `justify="center"` to get the same result as `justifyContent` iirc